deeds-dapp-webapp/src/main/webapp/static/css/deeds.css

```
@@ -33,8 +33,7 @@ body, .theme--dark.v-application {
 }
 
 .mainPageLayout, .headerLayout {
-  width: 72rem;
-  max-width: 100%;
+  width: 100%;
```

Keep the `max-width` in case it's larger than 100% to apply wrap and internal content resizing.

```suggestion
  width: 100%;
  max-width: 100%;
```

---

done!